### DISCUSSION TITLE
zh/docs/provider/storage/synology-nas

### DISCUSSION THREAD

**Discussion by giscus** [casdoor/casdoor#3107] [Docs comments]
# zh/docs/provider/storage/synology-nas

使用本地文件系统作为Casdoor的存储提供者

https://casdoor.org/zh/docs/provider/storage/synology-nas

**Comment by yhgroup** [comment]
上传nas返回408错误但前端提示上传成功了
{"error":{"code":408},"success":false}

**Reply by yhgroup** [reply]
nas启用家目录服务后才能上传成功，NewSynologyNasStorageProvider中指定了SharedFolder: "/home"
上传成功后但web端下载不了不清楚nas怎么设置！
